project/PythonPostBuild.scala

```diff
@@ -0,0 +1,59 @@
+import sbt._
+import sbt.Keys._
+import java.nio.file.{Files, Paths, Path, StandardCopyOption}
+
+object PythonPostBuild {
+  def processGeneratedFiles(
```

Comment:
please add a description of the transformation that are being done.

Reply:
actually you can just point to the readme that has the whole transformation process